### PR TITLE
Alina

### PR DESCRIPTION
**Pull request by Aleena-Khan12**

### CODE REVIEW

File: Admin Dashboard/src/App.jsx [hunk 1]
```diff
@@ -1,81 +1,63 @@
...
+                <Route path="/pricing/mileage" element={<Mileage />} />
+                
+              </Routes>  
+            </>
+          ) : (
+            // Show Login component if not authenticated
+            <Routes>
+            <Route path="/login" element={<Login onLogin={handleSuccessfulLogin} />} />
+            </Routes>    
+
```
Comment (Harisabdullah) [inline Admin Dashboard/src/App.jsx:55]: Remove unnecessary new lines

File: Admin Dashboard/src/components/Dashboard/OperatorDashboard.jsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+// import { useSelector } from 'react-redux/es/hooks/useSelector'
+// import { useDispatch } from 'react-redux'
+// import {Inc,Dec} from "../../states/reducers/index"
+
+
+
+export default function OperatorDashboard() {
+//     const currState=useSelector((state)=>state.number);
```
Comment (Harisabdullah) [inline Admin Dashboard/src/components/Dashboard/OperatorDashboard.jsx:15]: Remove uneccessary comments

**Review by Harisabdullah — COMMENTED** [review]
Good work. Just needs some changes

File: Admin Dashboard/src/components/Dashboard/OperatorDashboard.jsx (hunk 2 above)
Comment (Aleena-Khan12) [reply Admin Dashboard/src/components/Dashboard/OperatorDashboard.jsx:15]: ok

**Pull request merged**